kustomize.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: kustomize
+  version: 4.5.7
+  epoch: 0
+  description: Customization of kubernetes YAML configurations
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - bash
+      - curl
+      - ca-certificates-bundle
+      - busybox
+pipeline:
+  - runs: |
+      curl -sSLO https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh
+      bash install_kustomize.sh ${{package.version}}
```

Comment:
Can we build from source? It's unclear what this script will do.

Reply:
done @dlorenc